### PR TITLE
change text

### PR DESCRIPTION
**Pull request by maretamara**
poprawki 
* [ ] @korzonek 
* [x] @MagdaMalinowska 
* [x] @AdamMankowski

### CODE REVIEW

File: source/join-our-crew/content-writer.html.slim [hunk 1]
```diff
@@ -43,7 +43,7 @@
             br
               '
                 As the Content Writer, you will be responsible for writing
-                content that help our customers learn about things that
+                content that helps our customers learn about things that
                 matter to them e.g. development of successful web or mobile
```
Comment (AdamMankowski) [inline source/join-our-crew/content-writer.html.slim]: matters zamiast matter

Comment (AdamMankowski) [inline source/join-our-crew/content-writer.html.slim]: Linia 60 
You have passion for delivery of fresh, different,
powinno byc You have a passion for delivery of fresh, different,

Comment (AdamMankowski) [inline source/join-our-crew/content-writer.html.slim]: Linia 72:
traffic analytic tools
powinno byc traffic analysis tools

Comment (AdamMankowski) [inline source/join-our-crew/content-writer.html.slim]: Linia 73 
You played with tools like HubSpot, Buffer, Trello, etc
powinno byc You have played with tools like HubSpot, Buffer, Trello, etc

Comment (maretamara) [inline source/join-our-crew/content-writer.html.slim]: poprawione :)

Comment (maretamara) [inline source/join-our-crew/content-writer.html.slim]: poprawione

**Pull request merged**